tests/test_manifest.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+
+import pytest
+import yaml
+
+from bioimageio.spec import load_spec, utils
+
+MANIFEST_PATH = Path(__file__).parent / "../manifest.yaml"
+
+
+def pytest_generate_tests(metafunc):
```

Comment:
this generation of individual model tests from the manifest is outdated as we now have the `bioimageio.spec verify-bioimageio-manifest` command.

This test_manifest.py really does not test the manifest, but its content